existential-vs-universal-quantification/article.md

```diff
@@ -0,0 +1,194 @@
+# Existential vs universal quantification
+
+## Introduction
+This article gives brief information about one of the most common extension in Haskell projects - `ExplicitForAll`
+and syntax constructions that this extension brings to language.
+Stay tuned if you want to know how to create lists of differently typed variables, how to add complex constraints and more!
+
+From type theory we know about the existence of two quantifiers - `$\forall$` and `$\exists$`.
```

Comment:
But if you want to mention the origins of this (whether it be type theory or predicate logic or something else), it might be sensible to ask someone more knowledgeable on the topic to review the introduction.